charts/pega/values.yaml

```diff
@@ -328,6 +328,8 @@ cassandra:
 
 # DDS (external Cassandra) connection settings.
 # These settings should only be modified if you are using a custom Cassandra deployment.
+# To run Pega without Cassandra comment out or delete the following dds section and set
```

Comment:
slight tweak>>
To deploy Pega Platform without using Cassandra, comment out or delete the `dds` section and set
the `cassandra.enabled` property above to `false`.